manifests/charts/istio-cni/values.yaml

```diff
@@ -16,6 +16,7 @@ cni:
   cniBinDir: /opt/cni/bin
   cniConfDir: /etc/cni/net.d
   cniConfFileName: ""
+  cniFileWatchEnabled: true
```

Comment:
No need to add a value for this. I would prefer to just pass it through via istio operator.

Reply:
This is a little tricky to avoid a helm problem( [#3308](https://github.com/helm/helm/issues/3308)), it is quite complicated to use default value of a boolean variable.
So I set the default value in the `values.yaml`, and when migration, user could set value via istio operator like below.

```yaml
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
metadata:
  name: default
  namespace: istio-system
spec:
  components:
    cni:
      enabled: true
  values:
    cni:
      cniFileWatchEnabled: false
```

Reply:
I was actually hoping not to expose this as part of values at all, since this is only useful for one time migration. User could either pass in the env var via IstioOperator k8s overlay like:

```
apiVersion: install.istio.io/v1alpha1
kind: IstioOperator
metadata:
  name: default
  namespace: istio-system
spec:
  components:
    cni:
      enabled: true
      k8s:
       env:
       - name: CNI_FILE_WATCH_ENABLED
          value: "false"
```

Or even manually edit the daemonset to add that env var.